README.md

```diff
@@ -1 +1,247 @@
-# pfoos-full-stack-app
\ No newline at end of file
+# pfoos-full-stack-app
+
+## Index
+
+- [API Documentation](#api-documentation)
+  - [Contact Object](#contact-object)
+  - [Login Endpoint](#login-endpoint)
+  - [Add Contact Endpoint](#add-contact-endpoint)
+  - [Delete Contact Endpoint](#delete-contact-endpoint)
+  - [Update Contact Endpoint](#update-contact-endpoint)
+  - [Search Contacts Endpoint](#search-contacts-endpoint)
+
+## API Documentation
+
+### Contact Object
+
+JSON object used to transfer information about a single contact entry.
+
+- Parameters:
+
+  | Parameter   | Format | Description          |
+  | ----------- | ------ | -------------------- |
+  | `id`        | int    | Contact database key |
```

Comment:
**suggestion**: we should specify that this is the "contactId" like we do with "userId"

This goes for all other API Requests/Reponses that have a field "id"

Reply:
Good point, I'll make everything explicitly either "userId" or "contactId". That'll probably make the frontend code more consistent in the future as well.